lib/Extractor/Candidates.cpp

```diff
@@ -492,6 +497,11 @@ Inst *ExprBuilder::get(Value *V) {
   if (!E) {
     E = build(V);
   }
+  if (HarvestDemandedBits) {
+    E->DemandedBitsVal = APInt::getAllOnesValue(E->Width);
```

Comment:
is it safe to initialize DemandedBitsVal only conditionally? why not unconditionally initialize it?